packages/react-components/src/components/styles/VideoGallery.styles.ts

```diff
@@ -210,7 +210,8 @@ export const localVideoCameraCycleButtonStyles = (theme: Theme): IButtonStyles =
       margin: 0
     },
     flexContainer: {
-      paddingBottom: _pxToRem(8)
+      paddingBottom: _pxToRem(8),
+      height: 'none'
```

Comment:
what is `none` in relation to height? Is it 0? (as this component does have a height)
Does `unset` work instead?

Reply:
will give it a try! this was a weird one to find